internal/profile/resource_profile_test.go

```diff
@@ -233,6 +234,37 @@ func TestAccProfile_project(t *testing.T) {
 	})
 }
 
+func TestAccProfile_defaultProfile(t *testing.T) {
+	projectName := acctest.GenerateName(2, "-")
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			{
+				// Ensure default profile cannot be managed in default project.
+				Config:      testAccProfile_basic("default"),
+				ExpectError: regexp.MustCompile("Cannot import existing profile"),
```

Comment:
Is a regexp actually needed?

Reply:
Yes, this is the expected type of `ExpectError` field.

Reply:
Just attaching here:
```
field ExpectError *regexp.Regexp

ExpectError allows the construction of test cases that we expect to fail with an
error. The specified regexp must match against the error for the test to pass.

This functionality is only intended for provider-controlled error messaging. 
While in certain scenarios this can also catch testing logic error messages, 
those messages are not protected by compatibility promises.
```